features/step_definitions/custom_web_steps.rb

```diff
@@ -243,6 +245,10 @@ def take_screenshots_with_login
 end
 
 And /^I click close on all the popovers$/ do
+  page.execute_script("$('.popover .close')[0].click();")
+  page.should have_selector(".popover", count: 2)
+  page.execute_script("$('.popover .close')[0].click();")
+  page.should have_selector(".popover", count: 1)
```

Comment:
There's none, it's just the preferred style now so RSpec can eventually stop adding a method to every object in existence.